src/garage/tf/algos/ddpg.py

```diff
@@ -213,6 +216,7 @@ def init_opt(self):
             self.f_update_target = f_update_target
 
     def __getstate__(self):
+        """Object.__getstate__."""
```

Comment:
These are private functions, so they shouldn't need docstrings?

Reply:
As per `D105`, doc strings are required for magic methods.

Reply:
Huh. That feels pretty dumb. We can discuss changing that at a later date though.